File: src/main/java/de/vincentschweiger/phantomclient/cosmetics/CosmeticUtils.java
package de.vincentschweiger.phantomclient.cosmetics;

import de.vincentschweiger.phantomclient.server.ServerInfoProvider;
import net.minecraft.client.network.AbstractClientPlayerEntity;

public class CosmeticUtils  {
    public static boolean checkClientUser(AbstractClientPlayerEntity player) {
        return ServerInfoProvider.isClientPlayer(player.getGameProfile().getId());
    }
}

File: src/main/kotlin/de/vincentschweiger/phantomclient/cosmetics/CosmeticUtils.kt
package de.vincentschweiger.phantomclient.cosmetics

import de.vincentschweiger.phantomclient.socket.ServerInfoProvider.isClientPlayer
import net.minecraft.client.network.AbstractClientPlayerEntity

object CosmeticUtils {
    fun checkClientUser(player: AbstractClientPlayerEntity): Boolean {
        return isClientPlayer(player.gameProfile.id)
    }
}